code/obj/item/building_materials.dm

```diff
@@ -278,6 +278,7 @@ MATERIAL
 				L["remetal"] = "Remove Reinforcement"
 		if (src?.material.mat_id == "cardboard")
 			L["c_box"] = "Cardboard Box (2 Sheets)"
+			L["c_rcd"] = "Realistic Cardboard Device (10 sheets)"
```

Comment:
Disagree with this being made from cardboard. Should probably be one of those "hacked general fabricator" things if anything, made from some amount of the usual materials.

Reply:
it being difficult to make would make it incredibly unbalanced, as the only way to get cardboard material to reload it with is with cardboard boxes; I have not so far found a way to both allow this "RCD" to be good at building creatively or repairing an area, and yet bad at just filling an area with walls. Allowing for the cardboard RCD to be created from these walls(once broken into sheets) means that anyone with a welder and a wrench would be able to clean up the mess someone made with a cardboard RCD.

Reply:
"Print it from a general fabricator" is not "difficult to make".